services/src/main/java/com/linkedin/kmf/common/Utils.java

```diff
@@ -27,6 +30,9 @@
 public class Utils {
   private static final Logger LOG = LoggerFactory.getLogger(Utils.class);
 
+  private static final int ZK_CONNECTION_TIMEOUT = 30_000;
```

Comment:
Elsewhere it was the literal 30000. When I made it a constant I used the 30_000 format because I find it more readable than 30000.  I was not sure of the units for these parameters so I did not put that in the name of the constant.  I will add them to the name of the constant.